gio/src/file.rs

```diff
@@ -1111,6 +1113,98 @@ pub trait FileExtManual: IsA<File> + Sized {
 
         (fut, Box::pin(receiver))
     }
+    #[doc(alias = "g_file_set_attribute")]
+    fn set_attribute<T>(
+        &self,
+        attribute: &str,
+        type_: FileAttributeType,
+        value: FileAttributeTypeValue<T>,
```

Comment:
There should be type-system matching from the file attribute to the value type probably. If you e.g. pass a u32 where a u64 was expected this will read random stack memory.